.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 100
+}
```

Comment:
I recommend use this too:
```
  trailingComma: 'es5',
  arrowParens: 'always',
```
- Less changes in review
- Use parens for consistently output
- Also i recommend don't use printWidth: `100` with spaces, it is can generate unreadable code